ksql-engine/src/test/java/io/confluent/ksql/planner/plan/AggregateNodeTest.java

```diff
@@ -569,6 +570,14 @@ private KTable mapValues(final ValueMapper mapper) {
         mapValues.put(mapper, table);
         return table.createProxy();
       }
+
+      Stream<FakeKTable> tables() {
```

Comment:
what is going on here?

Reply:
This is needed to find the SelectMapper the test shouldUseConsistentOrderInPostAggSelectMapper needs.